QUARKUS-5622.md

```diff
@@ -0,0 +1,43 @@
+# Quarkus-5622 Support @PermissionAllowed for @BeanParam parameter
+
+Jira: https://issues.redhat.com/browse/QUARKUS-5622
+
+Upstream PR: https://github.com/quarkusio/quarkus/pull/43353/ that close the issue https://github.com/quarkusio/quarkus/issues/43231  
+
+## Scope of testing
+
+ Verifying the functionality that allows Quarkus to pass specific fields from `@BeanParam` parameters into custom `java.security.Permission` constructor 
+parameters.
+The testing will focus on:
+- Verify that specific fields from `@BeanParam` can be correctly referenced and passed to custom permission constructors.
+- Ensure permissions constructor parameters are matched based on secured method parameter names.
+- Test nested structures with multiple levels of depth
```

Comment:
It's not fully clear to me what this mean. Like having `SimpleBeanParam` which will extend `BaseBeanParam` for example? Or something different?

Reply:
It's about accessing fields in objects that are themselves fields of the `BeanParam`, I will try to clarify it.